tools.Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM python:3.8.9-slim 
+LABEL authors="phil.ewels@scilifelab.se,erik.danielsson@scilifelab.se" \
+      description="Docker image containing requirements for the nfcore tools"
+
+# Do not pick up python packages from $HOME
+ENV PYTHONNUSERSITE=1
+
+# Update pip to latest version
+RUN python -m pip install --upgrade pip
+
+# Install dependencies
+COPY requirements.txt requirements.txt
+RUN python -m pip install -r requirements.txt
+
+# Install Nextflow dependencies
+RUN apt-get update \
+      && apt-get upgrade -y \
+      && apt-get install -y git \
+      && apt-get install -y curl
+
+# Create man dir required for Java installation 
+# and install Java
+RUN mkdir -p /usr/share/man/man1 \
+      && apt-get install -y  openjdk-11-jre \
+      && apt-get clean -y && rm -rf /var/lib/apt/lists/*
+
+# Install Nextflow
+RUN curl -s https://get.nextflow.io | bash \
```

Comment:
Only improvement I can see here, is to al least specify an ENV variable for the Nextflow version.
That way we could build a tools image with a specific Nextflow version, with something like:
```
docker build -t nfcore/tools:specific_NXF . --build-arg NXF_VER=${NXF_VER}
```

Reply:
Sounds good! I guess that if it's not set then it will be empty and have the same behaviour as now anyway? So shouldn't be an issue. 👍🏻

Reply:
Wait, do we need to change anything in the `Dockerfile` for this? Your command should just set the env var and then the nextflow installer will see it, right? So just needs documentation?

Reply:
@ewels I made suggestion a couple of lines up, this is what needs to be added:
```
ARG NXF_VER=""
ENV NXF_VER ${NXF_VER}
```

Reply:
and yeah docs could be good

Reply:
Ah sorry I missed this, looking at the overview instead of code